payload.go

```diff
@@ -131,7 +131,7 @@ func NewKey(keyId string, cert *x509.Certificate, chain []*x509.Certificate) (*K
 		ret.N = base64.RawURLEncoding.EncodeToString(rsaPubKey.N.Bytes())
 
 		buf := new(bytes.Buffer)
-		err := binary.Write(buf, binary.BigEndian, ret.E)
+		err := binary.Write(buf, binary.BigEndian, int32(rsaPubKey.E))
```

Comment:
no test?

Reply:
They are used in tests in the `ziti` project now, but I added tests here too.